features/configuration/run_all_when_everything_filtered.feature

```diff
@@ -1,78 +0,0 @@
-Feature: run all when everything filtered
```

Comment:
People searching for it will end up at relish (it tends to be higher up in results than our docs) and if we remove it they'll end up at an older version page with no idea it's deprecated.